### PR TITLE
[APF-969] fix hovering over unconnected nodes

### PR DESCRIPTION
**Pull request by kborkowska**
It fixes the bug, which prevents nodes that don't have edges connected to them  form having hovered over actions fired, i.e. if cursor hovered over such a node it wouldn't change colour or show a tool-tip. 
Before:  
hilight_tooltip_connected_nodes
After:
hilight_tooltip_all_nodes

Solution was taken form: https://github.com/visjs/vis-network/commit/3dc7009d0921e2af72fa9d3ea91df77343b23fb6

### CODE REVIEW

File: lib/network/modules/SelectionHandler.js [hunk 1]
```diff
@@ -656,8 +656,9 @@ class SelectionHandler {
       const hoveredEdgesCount = Object.keys(this.hoverObj.edges).length;
       const hoveredNodesCount = Object.keys(this.hoverObj.nodes).length;
       const newOnlyHoveredEdge = object instanceof Edge && hoveredEdgesCount === 0 && hoveredNodesCount === 0;
+	  const newOnlyHoveredNode = object instanceof Node && hoveredEdgesCount === 0 && hoveredNodesCount === 0;
```
Comment (rafael-saback) [inline lib/network/modules/SelectionHandler.js]: This line is not properly indented. It seems to have a `tab` and because of that it looks strange.

**Comment by rafael-saback** [comment]
Good job @kborkowska! :slightly_smiling_face: 

@Gelio, this is a bug that we noticed recently and Karolina has found the solution in the new community-supported vis repo. It's a quite fresh bugfix, [it was merged 4 days ago](https://github.com/visjs/vis-network/pull/28). It seems that the people supporting the repo are quite active recently, it's already on version 5.1.1! (we're on 4.21.0)

If I remember well, we'll keep doing the patching as before. Let me know if anything has changed since then.

**Comment by Gelio** [comment]
@rafael-saback Nope, nothing has changed :smile: Use https://github.com/Juniper/contrail-ui-third-party-cache as usual :smile:

**Review by rafael-saback — APPROVED** [review]
Good job! Now hovering is working for unlinked nodes as well :slightly_smiling_face:

**Review by ghost — APPROVED** [review]

**Pull request merged**